reconcile/utils/terrascript_client.py

```diff
@@ -3299,6 +3299,32 @@ def populate_tf_resource_elasticsearch(self, resource, namespace_info):
         if self._multiregion_account_(account):
             es_values['provider'] = 'aws.' + region
 
+        advanced_security_options = values.get('advanced_security_options', {})
+        if advanced_security_options:
```

Comment:
Not mandatory, just small suggestion: The function is quite humongous - would it make sense to migrate this section to a new function, like:

```
if advanced_security_options:
    es_values['advanced_security_options'] = extract_advanced_security_options(advanced_security_options)
    
...

def extract_advanced_security_options(Dict options) -> Dict:  # Not sure if the types could be narrowed down a bit more here maybe
    pass
```

Reply:
it is humongouns, indeed... I'll follow your advise

Reply:
Fixed in 6c9a83d